rcsbsearchapi/search.py

```diff
@@ -959,15 +959,17 @@ class Session(Iterable[str]):
     return_type: ReturnType
     start: int
     rows: int
+    computational: bool
 
     def __init__(
-        self, query: Query, return_type: ReturnType = "entry", rows: int = 10000
+        self, query: Query, return_type: ReturnType = "entry", rows: int = 10000, computational: bool = False  # parameter for computed model inclusion
     ):
         self.query_id = Session.make_uuid()
         self.query = query.assign_ids()
         self.return_type = return_type
         self.start = 0
         self.rows = rows
+        self.computational = computational
```

Comment:
I do like the idea of having the users simply use a boolean to indicate that they want to include CSMs in their searches, but I worry this hides the option of `"results_content_type"`. Also, in the way it's implemented here, it's not possible for the user to search for *only* CSMs. 

Instead of doing it this way, I would recommend making this parameter `self.results_content_type`, which would by default be set to `["experimental"]`. Then, if the user wants to change it, they could either set it to be both (`["experimental", "computational"]`) or one or the other (e.g., `["computational"]`).

Then, you could also create a `Literal` type checking parameter for these options.

What do you think @brindakv?

Reply:
Agree with @piehld. We should allow for users to search experimental only, CSM only, or both. The RCSB.org frontend logic is specific for a different reason. We don't need to adopt that here.